Tests/TestUtilities/TestLogger.cs

```diff
@@ -101,6 +101,11 @@ public void AssertErrorLogged(string expected)
             Errors.Should().Contain(expected);
         }
 
+        public void AssertWarningLogged(string expected)
```

Comment:
Where is this used? VS doesn't find me a reference.

Reply:
In fact i added this for a test, but since the method is mocked warning were not working as expected, forgot to remove it :)